ldap/servers/slapd/back-ldbm/db-mdb/mdb_instance.c

```diff
@@ -377,18 +378,21 @@ dbmdb_open_all_files(dbmdb_ctx_t *ctx, backend *be)
         ctx = MDB_CONFIG(li);
     }
     ctxflags = ctx->readonly ? MDB_RDONLY: MDB_CREATE;
-    if (be) {
-        inst = (ldbm_instance *)be->be_instance_info;
-        /*
-         * Must ensure that vlv search list are initialized
-         * before calling vlv_getindices.
-         * and vlv_init must not be called while dbis_lock is held.
+    if (be && be->vlvSearchList_lock == NULL) {
```

Comment:
vlvSearchList_lock is here a flag to indicated that vlv_init was called.
You may use a function or a define, that test that lock, with a  more meaningful name (i.e. vlv_initialized() )

Reply:
will do so.